zipline/testing/core.py

```diff
@@ -1406,3 +1168,177 @@ def make_test_handler(testcase, *args, **kwargs):
     handler = TestHandler(*args, **kwargs)
     testcase.addCleanup(handler.close)
     return handler
+
+
+def write_compressed(path, content):
+    """
+    Write a compressed (gzipped) file to `path`.
+    """
+    with gzip.open(path, 'wb') as f:
+        f.write(content)
+
+
+def read_compressed(path):
+    """
+    Write a compressed (gzipped) file from `path`.
+    """
+    with gzip.open(path, 'rb') as f:
+        return f.read()
+
+
+zipline_git_root = abspath(
+    join(realpath(dirname(__file__)), '..', '..'),
+)
+
+
+def test_resource_path(*path_parts):
+    return os.path.join(zipline_git_root, 'tests', 'resources', *path_parts)
+
+
+@contextmanager
+def patch_os_environment(remove=None, **values):
```

Comment:
This was added to test the new feature that this change started on. We will use this soon.